pkg/api/persistence/persistence_test.go

```diff
@@ -619,6 +733,47 @@ func TestPersistence_GetScanChecks(t *testing.T) {
 	}
 }
 
+func TestPersistence_GetScanStatus(t *testing.T) {
+	loadFixtures(t)
+	tests := []struct {
+		name    string
+		scanID  uuid.UUID
+		want    api.Scan
+		wantErr bool
+	}{
+		{
+			name:   "ReturnsStatsHappyPath",
+			scanID: UUIDFromString(fixtureScans["Scan1"].ID),
+			want: api.Scan{
+				ID:             UUIDFromString(fixtureScans["Scan1"].ID),
+				Status:         util.Str2Ptr("RUNNING"),
+				CheckCount:     util.Int2Ptr(1),
+				ChecksFinished: util.Int2Ptr(2),
```

Comment:
I believe the count and finished values are swaped due to the error on the function implementation mentioned on previous comment.

Reply:
Absolutely thanks!!